Tools/gazebo_sitl_multiple_run.sh

```diff
@@ -47,27 +53,31 @@ then
 	exit 1
 fi
 
-while getopts n:m:w:s: option
+while getopts n:m:w:s:t: option
 do
 	case "${option}"
 	in
 		n) NUM_VEHICLES=${OPTARG};;
 		m) VEHICLE_MODEL=${OPTARG};;
 		w) WORLD=${OPTARG};;
 		s) SCRIPT=${OPTARG};;
+		t) TARGET=${OPTARG};;
 	esac
 done
 
 num_vehicles=${NUM_VEHICLES:=3}
-export PX4_SIM_MODEL=${VEHICLE_MODEL:=iris}
 world=${WORLD:=empty}
-echo ${SCRIPT}
-
+target=${TARGET:=default}
+if [ $TARGET == "rtps" ] && [ -z $VEHICLE_MODEL ] ; then
+	VEHICLE_MODEL=iris_rtps
+fi
+export PX4_SIM_MODEL=${VEHICLE_MODEL:=iris}
 
+echo ${SCRIPT}
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 src_path="$SCRIPT_DIR/.."
 
-build_path=${src_path}/build/px4_sitl_default
+build_path=${src_path}/build/px4_sitl_${target}
```

Comment:
Shouldn't the target be `px4_sitl_default` or `px4_sitl_rtps`?

Reply:
Umm... I don't understand what you said exactly. But, we can set any target using option "t".

Reply:
The target is `px4_sitl_default` or `px4_sitl_rtps` not `default` or `rtps`. therefore I think it is better we define the target clearly. Otherwise we are introducing another layer of configurations

Reply:
oops! I missed it. I  update it. Done